paddle/fluid/operators/dropout_op.cu

```diff
@@ -66,15 +56,27 @@ class GPUDropoutKernel : public framework::OpKernel<T> {
       std::random_device rnd;
       int seed =
           context.Attr<bool>("fix_seed") ? context.Attr<int>("seed") : rnd();
+      std::minstd_rand engine;
```

Comment:
In the future, we want to have a unified place for generating random number for all palldle (unify the cpu kernel)

Reply:
good idea.